relisten/realm/models/source_track.ts

```diff
@@ -87,6 +90,22 @@ export class SourceTrack
     return this._humanizedDuration;
   }
 
+  // eslint-disable-next-line getter-return
+  get artist(): Artist | null | undefined {
+    if (realm) {
+      const artist = realm.objectForPrimaryKey(Artist, this.artistUuid);
+      return artist;
+    }
+  }
+
+  // eslint-disable-next-line getter-return
+  get source(): Source | null | undefined {
+    if (realm) {
+      const source = realm.objectForPrimaryKey(Source, this.sourceUuid);
+      return source;
+    }
+  }
+
```

Comment:
yeah i think avoiding these would be desirable...they could be disastrous performance wise if they get misused